Src/libCZI/CZIReader.cpp

```diff
@@ -23,15 +23,21 @@ CCZIReader::~CCZIReader()
 {
 }
 
-/*virtual */void CCZIReader::Open(std::shared_ptr<libCZI::IStream> stream)
+/*virtual */void CCZIReader::Open(const std::shared_ptr<libCZI::IStream>& stream, const ICZIReader::OpenOptions* options)
```

Comment:
Isn't this a backward incompatible change if we now require a caller to pass the nullptr for `options` to obtain the "old" behavior?

Reply:
I'd argue no, because it is stated "for deciding about whether it is a breaking change, we are considering the source-code-distribution viewpoint (not binary-compatibility)", and from this point of view, the default argument should do the trick. At least - for almost all cases.